src/amo/components/LanguagePicker.scss

```diff
@@ -0,0 +1,20 @@
+@import "~core/css/inc/vars";
+
+$selector-padding: 8px;
+
+.LanguagePicker-header {
+  margin: 0 0 5px;
+}
+
+.LanguagePicker-selector {
+  appearance: none;
+  background: rgba(255, 255, 255, 0.25) url('../img/icons/triangle-down.svg') no-repeat calc(100% - #{$selector-padding}) 50%;
```

Comment:
Lint actually yelled at me for doing a line break IIRC. I'll try again and possibly tweak the rule, because I agree.